eng/publish/publish.yml

```diff
@@ -0,0 +1,281 @@
+# This is our package-publishing pipeline.
+# When executed, it automatically publishes the output of the 'official pipeline' (the nupkgs) to our internal ADO feed.
+# It may optionally also publish the packages to NuGet, but that is gated behind a manual approval.
+
+trigger: none # only trigger is manual
+pr: none # only trigger is manual
+
+resources:
+    repositories:
+        - repository: 1es
+          type: git
+          name: 1ESPipelineTemplates/1ESPipelineTemplates
+          ref: refs/tags/release
+        - repository: eng
+          type: git
+          name: engineering
+          ref: refs/tags/release
+
+    pipelines:
+    - pipeline: officialPipeline # Reference to the pipeline to be used as an artifact source
+      source: 'durabletask-dotnet.official'
+
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1es
+  parameters:

```

Comment:
Consider adding:

``` suggestion
  parameters:
    featureFlags:
      sdlNugetPathOptimization: true
```

Reply:
Can you tell me a bit more about what this does? I'd like to include some context as an inline comment

Reply:
I am not sure what it all does _exactly_. SDL is something 1ES team controls and their nuget release docs recommended adding it. Its probably fine to omit though